alert-policies/sonarqube/ActiveDatabaseConnections.yml

```diff
@@ -1,16 +1,16 @@
 # Name of the alert
-name: Active Database Connections
+name: Percentage of Active Database Connections
 
 # Description and details
 description: |+
-  This alert is triggered if the database connections are below 1 for 5 minutes.
+  This alert is triggered if the percentage of database connections are exceeds 90% for 5 minutes.
```

Comment:
```suggestion
  This alert is triggered if the percentage of database connections exceeds 90% for 5 minutes.
```

Reply:
Hi @mdumpati ,
As per your suggestion, I have made changes. Kindly review it.

Thanks.